Forms/ThumpNet.cs

```diff
@@ -32,20 +34,26 @@ public ThumpNet(ThumperModdingTool _ThumperModdingTool)
             countRequestBody.offset = 0;
             countRequestBody.limit = 0;
 
-            //urlBase = "http://127.0.0.2";
-            //Console.WriteLine("Attempting local thumpnet connection");
-            //var responseObject = MakeLevelPostRequest(countRequestBody);
+            LevelResponseBody? responseObject = null;
+            bool attemptLocalConnections = true; // Make sure this is false for public releases
```

Comment:
Noticed this was commented out in master, just added a boolean flag so easily swap for indev, make sure this is false for public releases